docs/concepts/tokens/decentralized-exchange/automated-market-makers.md

```diff
@@ -57,6 +57,14 @@ Tokens with different issuers are considered different assets. This means that t
 
 When the flow of funds between the two assets in a pool is relatively active and balanced, the fees provide a source of passive income for liquidity providers. However, when the relative price between the assets shifts, liquidity providers can take a loss on the [currency risk](https://www.investopedia.com/terms/c/currencyrisk.asp).
 
+### DEX Interaction
+
+AMMs are integrated with the central limit order book (CLOB)-based DEX to enhance liquidity on the XRPL. Offers and payments are automatically optimized to determine whether swapping within a liquidity pool, through the order book, or both, provides the best rate and executes accordingly. This ensures that transactions use the most efficient path for trades, whether through offers on the DEX or through AMM pools, or a combination of the two.
+
+The diagram below illustrates how an offer interacts with other offers and AMM liquidity in the XRPL DEX.
+
+![Offer path through XRPL DEX.](/docs/img/amm-clob-diagram.png)
```

Comment:
You don't really need to mention that it's the XRPL when it's documentation on xrpl.org.

```suggestion
AMMs are integrated with the central limit order book (CLOB)-based DEX to enhance liquidity. Offers and payments are automatically optimized to determine whether swapping within a liquidity pool, through the order books, or both, provides the best rate and executes accordingly. This ensures that transactions use the most efficient path for trades, whether through offers on the DEX or through AMM pools, or a combination of the two.

The diagram below illustrates how an offer interacts with other offers and AMM liquidity in the DEX.

![Offer path through DEX.](/docs/img/amm-clob-diagram.png)
```